packages/next/src/client/components/react-dev-overlay/utils/use-websocket.ts

```diff
@@ -115,3 +121,29 @@ export function useWebsocketPing(
     return () => clearInterval(interval)
   }, [tree, sendMessage])
 }
+
+export function reportHmrLatency(
```

Comment:
Could we use the function from packages/next/src/client/components/react-dev-overlay/utils/report-hmr-latency.ts here to avoid duplication?

Reply:
Whoops, yes, that was the intention, I accidentally duplicated this in my many rebases of these commits.